app/posts/register-of-training-providers/2025-03-04-reorganising-the-layout-of-the-provider-pages.md

```diff
@@ -0,0 +1,257 @@
+screenshots:
+  items:
+    - text: Provider details
+      src: secondary-navigation--provider-details.png
+    - text: Provider accreditations
+      src: secondary-navigation--provider-accreditations.png
+    - text: Provider accreditations - empty
+      src: secondary-navigation--provider-accreditations-empty.png
+    - text: Provider addresses
+      src: secondary-navigation--provider-addresses.png
+    - text: Provider addresses - empty
+      src: secondary-navigation--provider-addresses-empty.png
+    - text: Provider contacts
+      src: secondary-navigation--provider-contacts.png
+    - text: Provider contacts - empty
+      src: secondary-navigation--provider-contacts-empty.png
+    - text: Provider partnerships
+      src: secondary-navigation--provider-partnerships.png
+    - text: Provider partnerships - empty
+      src: secondary-navigation--provider-partnerships-empty.png
+---
+
+Previously, each training provider’s details were presented on a single, lengthy page. While this gave users access to all information at once, it posed significant usability issues:
+
+- information overload – users had to scroll extensively to find specific details, which could feel overwhelming and frustrating, particularly on mobile devices
+
+- discoverability issues – with all content grouped onto a single page, critical information could easily be overlooked or harder to reference quickly
+
+- interaction constraints – a single long page made it impossible to implement interactions such as pagination, sorting, filtering, or search within individual sections
+
+We introduced a secondary (horizontal) navigation to resolve these issues by clearly dividing provider information into logical sections. This offers several advantages:
+
+- improved findability – users can directly navigate to specific sections, making information retrieval quicker and more intuitive
+
+- enhanced readability – breaking content into smaller, manageable sections prevents cognitive overload and enhances clarity
+
+- scalability and flexibility – a structured navigation pattern allows for quickly incorporating future changes or new sections without significantly impacting the existing user experience
+
+- supports richer interactions – separate sections remove previous interaction constraints, allowing for pagination, sorting, filtering, and targeted search within each section, improving users’ ability to manage, explore, and quickly locate specific items
+
+## What we changed
+
+We have updated the provider details pages to include:
+
+- secondary navigation
+- summary cards
+- pagination
+
+## How it works
+
+### Secondary navigation
+
+![Example secondary navigation](secondary-navigation-example.png "Example secondary navigation")
+
+The secondary navigation helps users navigate around the provider’s details. The design is based upon the [GOV.UK service navigation component](https://design-system.service.gov.uk/components/service-navigation/).
```

Comment:
Why [tabbed interfaced](https://inclusive-components.design/tabbed-interfaces/) was not considered?

https://design-system.service.gov.uk/components/tabs/

> The tabs component lets users navigate between related sections of content, displaying one section at a time.

Reply:
If you compare the example in the design system to this service, you will see that provider details, accreditations, addresses, contacts, and partnerships, whilst belonging to a provider and sharing similar summary card designs, do not directly relate to one another. Therefore, we use secondary navigation rather than tabs.

Tabs may be suitable for small amounts of related data; secondary navigation is better for significant amounts of unrelated data (i.e., each section is self-contained).

Generally, I avoid using tabs and accordions. You can often replace them with a good content hierarchy.

Reply:
Agreed, good content hierarchy cannot be replaced by rubbish tabs and accordions.